public/editor/scripts/editor/js/fc/bramble-ui-bridge.js

```diff
@@ -188,6 +188,31 @@ define(function(require) {
       analytics.event("Redo");
     });
 
+    // TouchEmulator
+    var _touchEmulatorEnabled = false;
+    function setTouchEmulator(value) {
+      if(value) {
+        bramble.enableTouchEmulator();
+      } else {
+        bramble.disableTouchEmulator();
+      }
+    }
+    $("#preview-pane-nav-touch-emulator").click(function() {
+      setTouchEmulator(!_touchEmulatorEnabled);
+      return false;
```

Comment:
why the `return false`? Under HTML5 rules, that return statement should get ignored.